T'au Empire.cat

```diff
@@ -1,10 +1,7 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
-<catalogue id="c0bb-c0cd-a715-99c6" name="T&apos;au Empire" revision="71" battleScribeVersion="2.03" authorName="BSData Developers" authorContact="@Amis92" authorUrl="https://discord.gg/KqPVhds" library="false" gameSystemId="28ec-711c-d87f-3aeb" gameSystemRevision="164" xmlns="http://www.battlescribe.net/schema/catalogueSchema">
+<catalogue id="c0bb-c0cd-a715-99c6" name="T&apos;au Empire" revision="72" battleScribeVersion="2.03" authorName="BSData Developers" authorContact="@Amis92 @Technoblazed" authorUrl="https://discord.gg/KqPVhds" library="false" gameSystemId="28ec-711c-d87f-3aeb" gameSystemRevision="168" xmlns="http://www.battlescribe.net/schema/catalogueSchema">
```

Comment:
I think the author-contact is supposed to be the Discord handle. Not sure though. ;D

```suggestion
<catalogue id="c0bb-c0cd-a715-99c6" name="T&apos;au Empire" revision="72" battleScribeVersion="2.03" authorName="BSData Developers" authorContact="@Amis92 @Techno" authorUrl="https://discord.gg/KqPVhds" library="false" gameSystemId="28ec-711c-d87f-3aeb" gameSystemRevision="168" xmlns="http://www.battlescribe.net/schema/catalogueSchema">
```